### ISSUE TITLE
Overlay and Form???

### ISSUE THREAD

**Issue by kodz1** [wet-boew/wet-boew#7956]
Not sure what to call this issue. It not breaking my code, just noticed it in the console log. When I delete an element using JavaScript from an form using this code:
```
var parent = delBtn.parentElement;
parent.parentElement.removeChild(parent);
```
Internet Explorer on Windows 7 throws a error: `Unable to get property 'getBoundingClientRect' of undefined or null reference`

This is the line of code: https://github.com/wet-boew/wet-boew/blob/2cffc785bcf44b0c6e528268a087552898d066da/src/plugins/overlay/overlay.js#L284

Weird part is I have no overlay on this page.

**Comment by LaurentGoderre** [comment]
That is one of the bad parts of WET4....all the event handlers are added always. It wouldn't be much of a change to fix this but it's high risk. Perhaps for a v4.1.....

**Comment by kodecount** [comment]
Opps. Yeah I just noticed that. This is very bad. At the very least should've been some sort of if statement to prevent the code under certain circumstances.

**Comment by kodecount** [comment]
But like you said hopefully the next major rework this should be looked at with all plugins.

**Comment by LaurentGoderre** [comment]
I addressed this in my prototype for a potential next major release but we can fix this in a v4.1 bu moving the logic in the inits.
